ncm-ceph/src/main/perl/Ceph/compare.pm

```diff
@@ -27,7 +27,16 @@ use Socket;
 use Storable qw(dclone);
 
 our $EC=LC::Exception::Context->new->will_store_all;
-
+Readonly::Hash my %ACTIONS => (
+            'compare_mon' => \&compare_mon,
```

Comment:
if the funtion name and the key always match, you can do (in the code)
```
$whatever = "add_gtw";
$self->$whatever(...)
```
it would be nicer if this is a 2 layer hashref, like
```perl
Readonly my $ACTIONS => {
    add => {
       gtw => \&add_gtw,
    }
}
```
(quotes for keys are not needed, unless the keys have eg a `-`, which perl would see in unqouted context as subtraction)